clarity_ext/dilution.py

```diff
@@ -220,9 +230,9 @@ def calculate_transfer_volumes(self):
                     transfer.buffer_volume *= scale_factor
                     transfer.scaled_up = True
             except (TypeError, ZeroDivisionError) as e:
-                transfer.sample_volume = None
-                transfer.buffer_volume = None
-                transfer.has_to_evaporate = None
+                transfer.sample_volume = 0
```

Comment:
Ah. Gotcha. So it is fine that it is set to `None`, or might it cause problems elsewhere? I'm thinking that having the values set to `0` might be ambiguous, because perhaps sometimes it might make sense that it actually *should be* zero? On the other hand perhaps that should be left as an exercise for the script writer?